doc/manual/cpp_base_library.md

```diff
@@ -48,51 +52,57 @@ if (nullptr != checkerBundleConfig)
+issue. Additional meta information can be added to an issue: file location (
+cFileLocation ), XML file location ( cXMLLocation ) or road information (
+cRoadLocation ). This information is relevant for the ReportModule, for
+example, to point out meaningful errors in a GUI. The report XML file has to
+follow the schema file doc/schema/xqar_report_format.xsd.
 
 ```c++
 myChecker->AddIssue(new cIssue("errorDescription", ERROR_LVL))
 myChecker->AddIssue(new cIssue("take a closer look", INFO_LVL, (cExtendedInformation*) new cFileLocation(3, 0, "This is row 3, column 0.")));
 ```
 
-Issues are always assigned to exactly one checker. A log level is assigned to an issue: if it
-represents an error: ERROR_LVL, if it's just an information: INFO_LVL.
+Issues are always assigned to exactly one checker. A log level is assigned to
+an issue: if it represents an error: ERROR_LVL, if it's just an information:
+INFO_LVL.
 
-All CheckerBundles must contain at least one Checker. Checkers can be created or queried using a
-factory pattern:
+All CheckerBundles must contain at least one Checker. Checkers can be created
+or queried using a factory pattern:
 
 ```c++
 auto pXSDCheckerBundle = new cCheckerBundle("myCheckerBundle");
 cChecker* pOSCVersionChecker = pXSDCheckerBundle->CreateChecker("xoscVersionChecker", "Checks the validity of an xosc version.");
 pOSCVersionChecker->AddIssue(new cIssue("Version not supported.", ERROR_LVL));
 ```
 
-In a ResultContainer, all CheckerBundles are added. The ResultContainer is responsible for reading,
-writing and managing the data.
+In a ResultContainer, all CheckerBundles are added. The ResultContainer is
+responsible for reading, writing and managing the data.
 
 ```c++
 pResultContainer->AddCheckerBundle(pXSDCheckerBundle);
 pResultContainer->WriteResults("resultFile"));
 ```
 
-All objects in a ResultContainer are managed automatically. A release of the objects is not
-necessary and can be done with a ``cResultContainer::Clear`` or in the destructor.
+All objects in a ResultContainer are managed automatically. A release of the
+objects is not necessary and can be done with a ``cResultContainer::Clear`` or
+in the destructor.
 
 ## Mapping `XMLLocation` to `FileLocation`
 
 - **XmlLocation**: Addressing in an XML file based on an XPath expression
 - **FileLocation**: A reference to a file with row and column
 
-Some issues include a XML location to indicate where the issue occurred. During result pooling, a
-file location is determined from the XML location and saved as well.
+Some issues include a XML location to indicate where the issue occurred. During
+result pooling, a file location is determined from the XML location and saved
+as well.
 
-The definition of the XML location is based on a XPath expression. Almost the complete XPath 2. 0
-Standard is supported.
+The definition of the XML location is based on a XPath expression. Almost the
+complete XPath 2. 0 Standard is supported.
```

Comment:
```
complete XPath 2. 0 Standard is supported.
```
There is a spourious space betwein the `2.` and `0`

(I don't know why github today does not allow me making suggestions)